File: BlotoutAnalytics/Models/TransactionData.m
//
//  TransactionData.m
//  BlotoutAnalyticsSDK
//
//  Created by Nitin Choudhary on 28/11/21.
//  Copyright © 2021 Blotout. All rights reserved.
//

#import "TransactionData.h"

@implementation TransactionData

-(instancetype)init
{
    self = [super init];
    self.transaction_tax = [NSNumber numberWithInt:0];
    self.transaction_total = [NSNumber numberWithInt:0];
    self.transaction_currency = @"";
    self.transaction_discount = [NSNumber numberWithInt:0];
    self.transaction_shipping = [NSNumber numberWithInt:0];
    
    return self;
}
@end

File: BlotoutAnalytics/Models/TransactionData.swift
//
//  TransactionData.swift
//  BlotoutAnalyticsSDK
//
//  Created by Poonam Tiwari on 21/03/22.
//

import Foundation

public class TransactionData: NSObject {
    
    public var transaction_id = "" //Required Parameter
    public var transaction_currency: String?
    public var transaction_total: NSNumber?
    public var transaction_discount: NSNumber?
    public var transaction_shipping: NSNumber?
    public var transaction_tax: NSNumber?
}
